src/index.html

```diff
@@ -22,6 +22,409 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__link"
+      >
+        <img
+          src="images/logo.png"
+          alt="header__logo"
```

Comment:
The `alt` attribute should be descriptive and reflect the content of the image. Instead of 'header__logo', a more descriptive text like 'Company logo' would be appropriate.